hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieDefaultTimeline.java

```diff
@@ -107,7 +107,7 @@ public HoodieTimeline filterCompletedAndCompactionInstants() {
   }
 
   @Override
-  public HoodieDefaultTimeline getCommitsAndCompactionTimeline() {
+  public HoodieDefaultTimeline getCommitsCompactionAndReplaceTimeline() {
```

Comment:
got it. so this is a rename. discard my earlier comment.

Reply:
how about `getWriteTimeline` simply

Reply:
sg